tests/integration/session/test_simple_storage.py

```diff
@@ -50,3 +61,35 @@ def test_azure_blob_storage():
         ),
     )
     storage.test_connection()
+
+
+@pytest.mark.skip(reason="Kerberos is not supported in CI")
```

Comment:
Is there a plan to support testing this in an automated way?

Reply:
Once possibility would be to maintain a single-node kerberized cluster for integration testing. The other would be to create a container for testing but it may be rather large to pull for PR tests.